.travis.yml

```diff
@@ -8,3 +8,4 @@ before_install: gem update bundler
 
 script:
   - rake test
+  - codeclimate-test-reporter
```

Comment:
It won't matter because of our build steps on Travis. But it won't hurt to add the `bundler exec`